### ISSUE TITLE
stack overflow exception with `heap`

### ISSUE THREAD

**Issue by IngeniariusSoftware** [cqfn/eo#492]
code
```java
[] > global
  heap 1024 > h
  h.pointer 0 8 > ptr

  [] > mem
    ptr.block > @
      8
      [b] (b.as-int > @)

  [] > main
    seq > @
      mem.write (5.as-bytes)
      stdout (sprintf "%d" mem)

  main > @
```

error
![image](https://user-images.githubusercontent.com/33983579/146346983-222f98c6-4312-467b-87d6-33e49e317f70.png)

**Comment by yegor256** [comment]
@rultor release, tag is `0.21.1`

**Comment by yegor256** [comment]
@IngeniariusSoftware it was a bug. fixed in 0e02e83. Try version 0.21.1, should work.

**Comment by 0crat** [comment]
Job `gh:cqfn/eo#492` is not assigned, can't get performer
